streams/src/test/java/org/apache/kafka/streams/integration/utils/EmbeddedKafkaCluster.java

```diff
@@ -314,8 +321,7 @@ private TopicsDeletedCondition(final Collection<String> topics) {
 
         @Override
         public boolean conditionMet() {
-            final Set<String> allTopics = new HashSet<>(JavaConverters.setAsJavaSetConverter(
-                brokers[0].kafkaServer().zkClient().getAllTopicsInCluster(false)).asJava());
+            final Set<String> allTopics = new HashSet<>(getAllTopics());
```

Comment:
It looks like we don’t need to make a copy here anymore

Reply:
fixed in new commit